bitnami/seaweedfs/README.md

```diff
@@ -1043,7 +1045,8 @@ enabled = false
 | `mariadb.primary.persistence.size`         | Persistent Volume size                                                                                                                                                                                                     | `8Gi`               |
 | `mariadb.primary.resourcesPreset`          | Set container resources according to one common preset (allowed values: none, nano, small, medium, large, xlarge, 2xlarge). This is ignored if primary.resources is set (primary.resources is recommended for production). | `micro`             |
 | `mariadb.primary.resources`                | Set container requests and limits for different resources like CPU or memory (essential for production workloads)                                                                                                          | `{}`                |
-| `externalDatabase.enabled`                 | Enable secret with external mariadb database credentials                                                                                                                                                                   | `false`             |
+| `externalDatabase.enabled`                 | Enable external database database support                                                                                                                                                                                  | `false`             |
+| `externalDatabase.dbms`                    | Database management system product (mariadb, postgresql)                                                                                                                                                                   | `mariadb`           |
```

Comment:
I don't know if `dbms` is the most appropriate name for the new parameter. Maybe "store" to be aligned with upstream terms? See:

- https://github.com/seaweedfs/seaweedfs/wiki/Filer-Stores

Reply:
Will update to 'store'